lib/main.dart

```diff
@@ -118,6 +115,7 @@ class MyApp extends StatelessWidget {
     );
   }
 
+  // TO DO - This can be optimised probably to remove html views.
```

Comment:
better to raise this in an issue vs using a TODO comment

Reply:
Good point.